src/javascript/app_2/App/Components/Elements/please_login_message.jsx

```diff
@@ -0,0 +1,31 @@
+import PropTypes    from 'prop-types';
+import React        from 'react';
+import { localize } from '../../../../_common/localize';
+
+const PleaseLoginMessage = ({ onLogin, onSignup }) => (
+    <div className='please-login'>
+        <div className='please-login__message'>
+            {localize('Please')}
+            &nbsp;
+            <a href='javascript:;' onClick={onLogin}>
+                {localize('log in')}
+            </a>
+            &nbsp;
+            {localize('or')}
+            &nbsp;
+            <a href='javascript:;' onClick={onSignup}>
+                {localize('sign up')}
+            </a>
+            &nbsp;
+            {localize('to view this page.')}
```

Comment:
couldn't think of a solution, where we pass a single string template to localize.

Reply:
why not use placeholders?

Reply:
we already have this string `'Please [_1]log in[_2] or [_3]sign up[_2] to view this page.'`

Reply:
@negarn yeah, but how to insert achors then? In binary-static we replace `[_n]` with strings, if we do the same here, then we would need to use `dangerouslySetInnerHtml`. And it would still not bind `onClick` callback correctly.

Reply:
true, maybe @morteza-binary or @binary-hamedanchi have some better ideas.

Reply:
Need a component/function to handle this. Breaking a sentence into several parts and translate them separately will lose the point in many languages.